src/braket/aws/aws_quantum_task.py

```diff
@@ -143,6 +145,10 @@ def create(
                 IR. If the IR supports inputs, the inputs will be updated with this value.
                 Default: {}.
 
+            native_gate_calibration (Optional[NativeGateCalibration]): A `NativeGateCalibration` for
+                user defined gate calibration.
```

Comment:
Why is this parameter called `native_gate_calibration` if it is just a user defined gate calibration?

Reply:
changed to gate_calibrations